recruitment/introduction.md

```diff
@@ -0,0 +1,26 @@
+## Holiday Extras
+
+#### Why a technical task?
+
+You might be wondering why we feel its important to test candidates. At Holiday Extras we use a variety of different technologies and we don't expect you to be aware of all of them! We've designed both of our tasks with this in mind.
+
+Below is a few points that we tend to look for when reviewing your submission:
+
+- Quality (is it functional?)
+- Structure (code layout, abstraction, modularisation)
+- Usability and design
+
+#### The tasks
+
+- [Build an API](developer-API-task.md)
+- [Flickr API](developer-flickr-task.md)
```

Comment:
A more explicit description for the front-end task? _Flickr API front-end_? _Flickr API consumer front-end_?